.teamcity/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
do we really need a git ignore here?

Reply:
Would you prefer that I add this to the parent directory's `.gitignore` file? Since this is only a problem when running `mvn compile` inside this directory, I thought it was appropriate to only store it in this directory.